geode-core/src/main/java/org/apache/geode/cache/execute/FunctionService.java

```diff
@@ -72,7 +72,7 @@ protected FunctionService(FunctionExecutionService functionExecutionService) {
    * @throws FunctionException if the region passed in is null
    * @since GemFire 6.0
    */
-  public static <I, O, A> Execution<I, O, A> onRegion(Region region) {
+  public static <IN, OUT, AGG> Execution<IN, OUT, AGG> onRegion(Region region) {
```

Comment:
This still doesn't conform to our style guide. The names either need to be a single character or CamelCase name followed by "T". In this case given the readability of a single letter isn't great I would ask for CamelCaseT. 

https://google.github.io/styleguide/javaguide.html#s5.2.8-type-variable-names

`public static <ArgumentT, ReturnT, AggregatorT> Execution<ArgumentT, ReturnT, AggregatorT> onRegion(Region region)`

Reply:
have to say this is the first time I see type variables defined as this. I've only seen examples like these:
`public interface Function<T, R> ` in java.util.function
`public class ObjectAssert<ACTUAL> extends AbstractObjectAssert<ObjectAssert<ACTUAL>, ACTUAL>` in org.assertj.core.api

Reply:
@pivotal-jbarrett Okay, I see your point. Changes be coming ...

Reply:
Geode adopted the Google Java Style Guide. It may differ from other style guides. If we would like to amend our guide we can. Until then we should follow the guide.